src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -69,3 +69,23 @@ dt {
     margin-top: 0;
   }
 }
+
+div.review {
+  border-bottom: 1px solid $color-gray-light;
+  .review-row {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    border-top: 1px solid $color-gray-light;
+    padding: 1.5rem;
+    padding-left: 0;
+  }
+  .review-row dd {
```

Comment:
You could nest these within the `.review-row {` selector on line 75 if you want 🙂